e2e-tests/src/main/java/gov/nasa/jpl/aerie/e2e/procedural/scheduling/procedures/DeleteBiteBananasGoal.java

```diff
@@ -0,0 +1,26 @@
+package gov.nasa.jpl.aerie.e2e.procedural.scheduling.procedures;
+
+import gov.nasa.ammos.aerie.procedural.scheduling.Goal;
+import gov.nasa.ammos.aerie.procedural.scheduling.annotations.SchedulingProcedure;
+import gov.nasa.ammos.aerie.procedural.scheduling.plan.DeletedAnchorStrategy;
+import gov.nasa.ammos.aerie.procedural.scheduling.plan.EditablePlan;
+import gov.nasa.ammos.aerie.procedural.timeline.payloads.activities.DirectiveStart;
+import gov.nasa.jpl.aerie.merlin.protocol.types.Duration;
+import gov.nasa.jpl.aerie.merlin.protocol.types.SerializedValue;
+import gov.nasa.jpl.aerie.types.ActivityDirectiveId;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Map;
+
+/**
+ * Deletes all Bite Bananas with extreme prejudice. Used to test that updated
+ * anchors are saved in the database properly.
+ */
+@SchedulingProcedure
+public record DeleteBiteBananasGoal() implements Goal {
+  @Override
+  public void run(@NotNull final EditablePlan plan) {
+    plan.directives("BiteBanana").forEach($ -> plan.delete($, DeletedAnchorStrategy.PreserveTree));
```

Comment:
I'd like to request tests that test the behavior of the other two `DeletedAnchorStrategy`s (`Error` and `Cascade`)

Reply:
Done, see bottom of databasedeletiontests